html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/SponsorEmail.pm

```diff
@@ -28,6 +28,16 @@ has_field 'allow_localdomain' =>
              help => 'Accept self-registration with email address from the local domain' },
   );
 
+has_field 'email_activation_timeout' =>
```

Comment:
You need to add the field `email_activation_timeout` to the SponsorEmailSource module as it doesn't seem to be there..